build.gradle

```diff
@@ -10,9 +10,6 @@ buildscript {
 
   dependencies {
     classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion"
-    classpath 'com.github.ben-manes:gradle-versions-plugin:0.20.0'
-    classpath 'com.vanniktech:gradle-code-quality-tools-plugin:0.16.0'
-    classpath 'com.vanniktech:gradle-android-junit-jacoco-plugin:0.16.0'
```

Comment:
those plugins have compatibility issues with the new kotlin. I understand its not ideal to fix a problem with a plugin removing it, but the upgrade would have not been possible otherwise. I tried the upgraded versions of those plugins but did not work.

I guess is more important to have a working upgraded lib for now